lua/mason-core/installer/managers/std.lua

```diff
@@ -108,7 +108,11 @@ end
 local function untar(rel_path)
     log.fmt_debug("std: untar %s", rel_path)
     local ctx = installer.context()
-    return ctx.spawn.tar({ "--no-same-owner", "-xvf", rel_path }):on_success(function()
+    local tar_cmd = ctx.spawn.tar
+    if platform.is.unix and not platform.is.linux then
+        tar_cmd = ctx.spawn.gtar
+    end
+    return tar_cmd({ "--no-same-owner", "-xvf", rel_path }):on_success(function()
```

Comment:
I think we can simplify this to simply prefer `gtar` if it exists and if not fall back to `tar`. That way we can avoid relying on broader platform detection and all the quirks it may incur. Something like the following ought to do it.

```lua
local a = require("mason-core.async")

-- …

function M.untar()
  -- …
  a.scheduler()
  local tar = vim.fn.executable("gtar") == 1 and "gtar" or "tar"
  return ctx.spawn[tar] { ... }
end
```

The `a.scheduler()` (also notice the additional import) is required because this function may execute outside Neovim's main loop, in which case `vim.fn` functions cannot be called.